docs/codeql/codeql-workshops/cpp/introduction/codeql-workshop-for-cpp.md

```diff
@@ -0,0 +1,62 @@
+# CodeQL workshop for C/C++
+
+This training workshop is a beginner level course which introduces CodeQL for C/C++.
+
+The course is divided into three sessions. Each session should take around 2 hours to complete.
+
+For each session, we have provided a series of videos. Watch each video in turn, and follow any instructions provided. Alternatively, you can complete the course by following the written instructions linked next to each section.
+
+The interactive videos are formatted to take up half the width of a HD screen, allowing you to follow along in your own VS Code window side-by-side with the video (for example, using [Split View](https://youtu.be/cYGrlEtqaVw) on Macs, or Win+Left, Win+Right hotkeys on Microsoft Windows). Some videos include exercises to be completed during the video. We recommend pausing the video when the exercise is read out, completing the exercise, then clicking play to continue the video.
+
+## Session 1
+
+_Estimated completion time: 2 hours_
+
+1. Introduction to CodeQL [[Video](https://drive.google.com/file/d/1DdyD5PGy6LeV2urnitiiwWeBVyNHy20G/view?usp=sharing)] _08:22_
```

Comment:
Are these public video links? I would rather have them removed, or hosted somewhere more permanent.

Reply:
could not find alt public links for the equivalent recordings